### PR TITLE
Add basic example for ios

### PR DESCRIPTION
**Pull request by alledy**
## Fix #
- Need to add a new basic example for ios

## Changes
- Made a new example named ClickThePaintings in Examples/Debugging
- Installed babel-plugin-module-resolver and add alias(`@: ./src`) in babel.config.js file
- Installed react-router-native for routing
- Implemented a function where click a painting and get brief information

## How Verified
N/A

### Notes
N/A

### References
N/A

### CODE REVIEW

**Comment by JeffGuKang** [comment]
Good. Do not forget to use capital letter for first character of the commit message.

**Review by JeffGuKang — CHANGES_REQUESTED** [review]
Good! Do not forget to use capital letter for first character of the commit message.

File: Examples/Debugging/ClickThePaintings/.gitignore [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# OSX
+#
+.DS_Store
+
+# Xcode
+#
+build/
+*.pbxuser
+!default.pbxuser
```
Comment (Duckuism) [inline Examples/Debugging/ClickThePaintings/.gitignore:9]: Why are you using this logical operator in here? Wanna know your intention.

File: Examples/Debugging/ClickThePaintings/src/App.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import React, {useState} from 'react';
+import {NativeRouter, Route} from 'react-router-native';
+import TopBar from '@/components/TopBar';
```
Comment (Duckuism) [inline Examples/Debugging/ClickThePaintings/src/App.js:3]: Thanks to you, I know something [new](https://stackoverflow.com/a/42753045)!

**Review by Duckuism — APPROVED** [review]
I was impressed with the good use of `react-router-native`

**Comment by JeffGuKang** [comment]
@alledy How about resolve requested changes?

**Comment by Duckuism** [comment]
@JeffGuKang She is traveling until next Wednesday. I think it will be resolved after next Wednesday. 😀

**Pull request merged**